modules/tracktion_engine/utilities/tracktion_EngineBehaviour.h

```diff
@@ -178,6 +178,8 @@ class EngineBehaviour
         1: loopRangeDefinesSubsequentRepetitions    // The first section is the whole sequence, subsequent repitions are determined by the loop range.
     */
     virtual int getDefaultLoopedSequenceType()                                      { return 0; }
+
+    virtual bool autoAddClipEdgeFades()                                             { return true; }
```

Comment:
This needs a doxygen comment describing what the function does. See the other methods here for an example of the detail level required. Basically just describe what returning true or false does here.

This also needs to be changed back to `false` by default for merging back in to develop to not change the behaviour for existing users.